src/main/java/no/nav/veilarbperson/controller/PdlController.java

```diff
@@ -12,16 +22,70 @@
 public class PdlController {
```

Comment:
Jeg tror jeg ville fjernet hele PDL controlleren og heller lage en PersonV2Controller som kun implementerer endepunktet for å hente ut personalia (dvs det endepunktet som henter masse info fra PDL)